src/enzo/Grid_ConvertToLibyt.C

```diff
@@ -52,11 +52,12 @@ void grid::ConvertToLibyt(int LocalGridID, int GlobalGridID, int ParentID, int l
      * In here, we'll happily do whatever we're told.
      *
      * */
-  
+    float grid_unit_length[MAX_DIMENSION] = {1.0, 1.0, 1.0};
     for (int i = 0; i < MAX_DIMENSION; i++) {
         GridInfo.grid_dimensions[i] = (this->GridEndIndex[i]) - (this->GridStartIndex[i]) + 1; // this is active dimension
         GridInfo.left_edge[i] = this->GridLeftEdge[i];
         GridInfo.right_edge[i] = this->GridRightEdge[i];
+        grid_unit_length[i] = (GridInfo.right_edge[i] - GridInfo.left_edge[i]) / GridInfo.grid_dimensions[i];
```

Comment:
Just confirming that you want to change this value from 1.0 to the dx of this grid. I believe you do because of the product below with the code particle mass (density). I'd recommend renaming it to indicate that it's the cell width and not the grid unit length.

Reply:
Thanks for the suggestion!